internal/db/changelog.go

```diff
@@ -13,10 +12,10 @@ import (
 )
 
 // Changelog - Function to pull all rules from db
-func (c *Client) GetAllChangelogEntries() ([]models.ChangelogEntry, error) {
+func (c *Client) GetAllChangelogEntries( offset int, limit int) ([]models.ChangelogEntry, error) {
 	rules := []models.ChangelogEntry{}
 
-	rows, err := c.Conn.Query("SELECT * FROM changelog ")
+	rows, err := c.Conn.Query("SELECT * FROM changelog LIMIT ?,?", offset, limit)

```

Comment:
it doesn't make sense to have an offset and limit for `GetAllChangelogEntries` method. Isn't that what `GetChangeLogEntries` is for?

Reply:
Limit and offset are used for pagination. That's why both have them.

Reply:
- `GetAllChangeLogEntries` should get ALL entries
 - `GetChangeLogEntries` should get entries with offset and limit
If you no longer have a need for `GetAllChangeLogEntries` than we should remove it, but as it stands both functions are doing the same thing

Reply:
resolving issue
 - `GetChangeLogEntries` takes in a ruleID but still requires offset and limit